src/chrome/content/rules/Lolagrove.com.xml

```diff
@@ -29,9 +27,6 @@
 	<securecookie host="^v3\.lolagrove\.com$" name=".+" />
 
 
-	<rule from="^http://lolagrove\.com/"
-		to="https://www.lolagrove.com/" />
-
```

Comment:
You should add a comment about the mismatch on the top, that it get not readded.